docs/internal/readme-internal.md

```diff
@@ -54,4 +54,7 @@ For external contributors:
    1. Clear description of the feature
    2. Test coverage
    3. Validation instructions
-   
\ No newline at end of file
+
+## Validations framework link to readme
+
+Path: docs/internal/input-validations-framework-readme.md
```

Comment:
Please add a link here, not just path text. It should be clickable.

Reply:
I did not know you could do that in a readme